modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/index/RocksDbSortedIndexStorage.java

```diff
@@ -202,14 +208,47 @@ public ByteBuffer next() {
 
                 this.hasNext = null;
 
-                return ByteBuffer.wrap(key).order(ORDER);
+                return mapper.apply(ByteBuffer.wrap(key).order(ORDER));
+            }
+
+            @Override
+            public @Nullable T peek() {
+                if (hasNext != null && !hasNext) {
+                    return null;
+                }
+
+                T next = null;
```

Comment:
This variable is redundant

Reply:
fix it

Reply:
Because you can write this code without it (it's just a nitpick, no big deal):
```
if (!it.isValid()) {
    RocksUtils.checkIterator(it);

    return null;
} else {
    return mapper.apply(ByteBuffer.wrap(it.key()).order(ORDER));
}
```

Reply:
Fix it